ripc-protocol-tcp/src/main/java/io/ripc/protocol/tcp/TcpHandler.java

```diff
@@ -0,0 +1,8 @@
+package io.ripc.protocol.tcp;
+
+import org.reactivestreams.Publisher;
+
+public interface TcpHandler<R, W> {
+
+    Publisher<Void> handle(Connection<R, W> connection);
```

Comment:
What does the returned `Publisher<Void>` signal to a `Subscriber` via `onComplete`? What happens if multiple handlers are added? Does each `Subscriber` receive an `onComplete` signal when the peer terminates?